tests/.infra/centos-ci/rc/rc_function_util.sh

```diff
@@ -7,19 +7,30 @@
 # http://www.eclipse.org/legal/epl-v10.html
 
 function prepareCustomResourceFile() {
+  RELEASE_VERSION=7.12.1
   echo "======== Patch custom-resource.yaml ========"
   cd /tmp
-  wget https://raw.githubusercontent.com/eclipse/che-operator/master/deploy/crds/org_v1_che_cr.yaml -O custom-resource.yaml
+  wget https://raw.githubusercontent.com/eclipse/che-operator/$RELEASE_VERSION/deploy/crds/org_v1_che_cr.yaml -O custom-resource.yaml
   setOpenShiftoAuth=$1 # sets the value of 'openShiftoAuth' parameter
   sed -i "s@openShiftoAuth: false@openShiftoAuth: $1@g" /tmp/custom-resource.yaml
   sed -i "s@server:@server:\n    customCheProperties:\n      CHE_LIMITS_USER_WORKSPACES_RUN_COUNT: '-1'@g" /tmp/custom-resource.yaml
   sed -i "s/customCheProperties:/customCheProperties:\n      CHE_WORKSPACE_AGENT_DEV_INACTIVE__STOP__TIMEOUT__MS: '300000'/" /tmp/custom-resource.yaml
-  sed -i "s@identityProviderImage: 'quay.io/eclipse/che-keycloak:nightly'@identityProviderImage: 'quay.io/eclipse/che-keycloak:$RELEASE_TAG'@g" /tmp/custom-resource.yaml
+  sed -i "s@identityProviderImage: 'quay.io/eclipse/che-keycloak:nightly'@identityProviderImage: 'quay.io/eclipse/che-keycloak:$RELEASE_VERSION'@g" /tmp/custom-resource.yaml
   sed -i "s@cheImage: ''@cheImage: 'quay.io/eclipse/che-server'@g" /tmp/custom-resource.yaml
-  sed -i "s@cheImageTag: 'nightly'@cheImageTag: '$RELEASE_TAG'@g" /tmp/custom-resource.yaml
+  sed -i "s@cheImageTag: 'nightly'@cheImageTag: '$RELEASE_VERSION'@g" /tmp/custom-resource.yaml
   sed -i "s@devfileRegistryImage: 'quay.io/eclipse/che-devfile-registry:nightly'@devfileRegistryImage: 'quay.io/eclipse/che-devfile-registry:$RELEASE_VERSION'@g" /tmp/custom-resource.yaml
   sed -i "s@pluginRegistryImage: 'quay.io/eclipse/che-plugin-registry:nightly'@pluginRegistryImage: 'quay.io/eclipse/che-plugin-registry:$RELEASE_VERSION'@g " /tmp/custom-resource.yaml
   sed -i "s@tlsSupport: true@tlsSupport: false@g" /tmp/custom-resource.yaml
   sed -i "s@identityProviderPassword: ''@identityProviderPassword: 'admin'@g" /tmp/custom-resource.yaml
   cat /tmp/custom-resource.yaml
 }
+
+function installReleaseCheCtl() {
```

Comment:
Also I think we should rewrite the rc tests to use the checluster CR patch mechanism of chectl.
That would allow us to just download the released chectl and everything should be in place (for example there will be no need of specifying [cheImageTag of the released version instead of nightly](https://github.com/eclipse/che/pull/16901/files#diff-805d68fc974c7a8f2b58ea2589a813d3R20))

Reply:
> Can you remind me why we can't just use bash <(curl -sL https://www.eclipse.org/che/chectl/)
There was limitation in CRW CCI regarding sudo rights, but on ci.centos it should be possible, IMHO

Reply:
> Also I think we should rewrite the rc tests to use the checluster CR patch mechanism of chectl.

Sounds reasonable. I will fix ci.centos scripts to use patch of CR instead of whole CR file..